myapp/backend/backend.py

```diff
@@ -34,12 +37,149 @@ def remove(self):
       return resp.deleted_count
 
 class User(Model):
-  client = pymongo.MongoClient("mongodb+srv://TeamProjAdmin:teamproject308@cluster0.3xlma.mongodb.net/TeamProj?retryWrites=true&w=majority&tls=true&tlsAllowInvalidCertificates=true")
-  # db = client.TeamProj
-  collection = client.TeamProj.TeamRoster
+  #client = pymongo.MongoClient("mongodb+srv://TeamProjAdmin:teamproject308@cluster0.3xlma.mongodb.net/TeamProj?retryWrites=true&w=majority&tls=true&tlsAllowInvalidCertificates=true")
   
+  def get_collection(name):
+    if(not User.check_for_collection(name)):
+      return None
+    client = pymongo.MongoClient("mongodb+srv://TeamProjAdmin:teamproject308@cluster0.3xlma.mongodb.net/TeamProj?retryWrites=true&w=majority&tls=true&tlsAllowInvalidCertificates=true")
```

Comment:
This looks good, however, we could probably section out parts that vary between each function and pass on the client to each function or separate them out.